src/main/java/network/kngsgaming/minecraft/EnchantEvents.java

```diff
@@ -221,16 +220,17 @@ public void onInventoryClick(InventoryClickEvent event){
     }
 
     //Check to see if this enchantment and level are within the config defined limits
+    //Unused?
```

Comment:
isVailidEnchantLevel() is called every time two items in the anvil have an enchantment with the same level. It basically checks to see if the two should combine to be a +1, or if that would take it past the limit in the config.

Reply:
Chrome's search option lied to me :| Sorry